modules/Hoot/ui/modals/importDataset.js

```diff
@@ -71,6 +69,17 @@ export default class ImportDataset {
         ];
     }
 
+    static importTypeMap() {
```

Comment:
chose to add here because import multiple is using the same api logic as import single anyways so figure this would be the root